### DISCUSSION TITLE
IN GitHub PROJECTS: How do you duplicate projects?

### DISCUSSION THREAD

**Discussion by samsonfsb** [community/community#10697] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a way to duplicate projects (e.g. so that they have the same column names/views/etc)?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there are you asking about projects or projects beta. For projects, find how to copy [here](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/copying-a-project-board).
